docs/client-side-usecases configuration.md

```diff
@@ -0,0 +1,105 @@
+last edit: 2016-11-02 11:13
+author: m4dz <code@m4dz.net>
+
+---
+
+
+Cozy MyAccounts Client Use-case
+===============================
+
+Abstract
+--------
+
+This document describes the context of using _use-cases_ in MyAccounts app, how they are related to the app and the stack, and how to customize them in hosting context.
+
+
+What are use-cases?
+-------------------
+
+A **use-case** is a way to sort and filter konnectors by an arbitrary denominator (e.g. _all konnectors that concerns billing_). It allow a user to browse available konnectors in a different way than browsing by categories. I allow the user to discover some konnectors by selecting first a _use-case_ (aka a scenario) that cover usages, and see what konnectors can be suggested.
+
+_MyAccounts_ can provides many _use-cases_, that can be configured, which mean a hoster can offer a different way to discover available konnectors (aka present different _use-cases_).
+
+
+How to define a use-case?
+-------------------------
+
+All _use-cases_ are declared in a manifest, built-in the client app, at `client/app/contexts/<context>/index.json`. Its architecture is defined ine the _Architecture of the manifest_ section below.
```

Comment:
My opinion is that, with an `index.json` file, we can safely do a `const ctx = require('app/contexts/<mycontext>')` to load a context because the require will try to load an `index` file by default, and I find this way very smart. But I'm open to any other suggestion :)